.github/workflows/publish.yml

```
@@ -20,3 +20,13 @@ jobs:
       with:
         command: publish
         args: --all-features
+    - name: slack notification (the job has failed)
+      if: failure()
+      run: |
+        curl -s -X POST -H "Content-Type: application/json" -d '${{ env.SLACK_TEXT }}' ${{ secrets.SLACK_CORE_TOOLS_TEAM_URL }}
+      env:
+        SLACK_TEXT: '{"attachments":[{
+        "pretext": "`transit_model Publish` requires your attention!",
+        "text":" :warning: Publish failed!","color":"#D00000",
```

minor, to be consistent:
```suggestion
        "text":" :warning: `Publish` failed!","color":"#D00000",
```

---

This format caused errors in CI workflow ( i don't know why). I prefere those like that if you want :)

---

OK, weird but all is consistent now.